app/mailers/comment_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class CommentMailer < ApplicationMailer
+  default from: "powercodegirls@gmail.com"
+  
+  def notification_new_comment(user_id, parent_id)
+
+    @user = User.find user_id
+    @parent_id = parent_id
+    @url  = "https://www.rubygamedev.com/internal_posts/#{@parent_id}"
+
+    mail(to: @user.email, subject: 'New comment from your post')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.